mage_ai/frontend/components/ContextMenu/index.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState, useCallback } from 'react';
+
+import BlockType from '@interfaces/BlockType';
+import FlyoutMenu from '@oracle/components/FlyoutMenu';
+import styled from 'styled-components';
+import { FileContextEnum } from '@components/FileBrowser';
+import { UNIT } from '@oracle/styles/units/spacing';
+import { HEADER_HEIGHT, HEADER_Z_INDEX } from '@components/constants';
+
+export type ContextMenuSharedProps = {
+  deleteBlockFile: (b: BlockType) => void;
+};
+
+export type ContextMenuProps = {
+  children: any;
+  areaRef: any;
+  type: ContextMenuEnum;
+  useContextItem: boolean;
```

Comment:
Rename this to something else that doesn't start with `use`. Can get confused with a react hook.